### ISSUE TITLE
Reload Pagination Data Filtering

### ISSUE THREAD

**Issue by dring1** [esvit/ng-table#465]
Hello, 

I have some external services/calls modifying the data in the current scope - I would like to reload the pagination bar - I am watching the data in the scope - when it changes I trigger

```
$scope.tableParams.reload();
```

Although this does not seem to reload the pagination bar, If I have 20 pages, modify the data down to 10 pages, the bar will continue to show 20 pages - anything beyond 10 pages will be a blank table.

Thank you.

**Comment by itsli** [comment]
Hi, I am having the same issue with the current version so I don't think this was fixed. I have dynamically loaded data that is changed when triggered by a checkbox.

The first time I load the table, everything is fine. However when I change the array's length and do a $scope.tableParams.reload();, strange things happen.

When I set the count to an integer in the parameters at making the table, it does not work at all. The pagination and the number of entries per page buttons disappears. The data is also wrong.

When I set the count to the length of the array at making the table, it mostly works, but the pagination is gone. If I change the number of entries per page and change it back the pagination comes back.

Thank you for your help!
